python/tests/table/test_metadata.py

```diff
@@ -515,7 +516,7 @@ def test_v1_write_metadata_for_v2():
     ]
     assert metadata_v2["default-spec-id"] == 0
     assert metadata_v2["last-partition-id"] == 1000
-    assert metadata_v2["sort-orders"] == [{"order-id": 0, "fields": []}]
+    assert metadata_v2["sort-orders"] == [{"fields": [], "order-id": 0}]
```

Comment:
Can some of these changes be separated into different PRs so that we don't carry them forward as extra changes while the REST catalog is reviewed?

Reply:
Unsure why this changed in the first place 🤔